backend/dal/Migrations/v01.11.00/Down/PostDown/01-NotificationTemplates.sql

```diff
@@ -0,0 +1,5 @@
+PRINT N'Update [NotificationTemplates]'
+
+UPDATE dbo.[NotificationTemplates]
```

Comment:
This doesn't undo the below notification template changes, does it need to?

Reply:
Doesn't need to in this case.